### PR TITLE
Update excel after 1.1.3 and masvs 1.2 release

### PR DESCRIPTION
**Pull request by A-AFTAHI**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

**Comment by sushi2k** [comment]
Happy to help. @A-AFTAHI let me know where I can support.

**Comment by commjoen** [comment]
: (: on line 33 we refer to testing keymanagement which only takls about reusing symmetric keys, not assymetric. Perhaps this is worth a separate issue.

**Comment by A-AFTAHI** [comment]
> 
> 
> : (: on line 33 for Andoird we refer to testing keymanagement which only talks about reusing symmetric keys, not assymetric. Perhaps this is worth a separate issue. (not fix here)

so the new issue well be MSTG related and not the excels since the excel only follows what's already there?

**Comment by commjoen** [comment]
> > : (: on line 33 for Andoird we refer to testing keymanagement which only talks about reusing symmetric keys, not assymetric. Perhaps this is worth a separate issue. (not fix here)
> 
> so the new issue well be MSTG related and not the excels since the excel only follows what's already there?

exactly, still reviewing though.

**Comment by commjoen** [comment]
https://github.com/OWASP/owasp-mstg/blob/1.1.3/Document/0x04h-Testing-Code-Quality.md#cross-site-scripting-flaws-mstg-arch-2-and-mstg-platform-2 does not really seem to cover mstg-arch2 right? @sushi2k maybe have an issue on it :D ?

File: Checklists/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+[![Creative Commons License](https://licensebuttons.net/l/by-sa/4.0/88x31.png)](https://creativecommons.org/licenses/by-sa/4.0/ "CC BY-SA 4.0")
+
+[![OWASP Flagship](https://img.shields.io/badge/owasp-flagship%20project-48A646.svg)](https://www.owasp.org/index.php/Category:OWASP_Project#tab=Project_Inventory)
+[![Build Status](https://travis-ci.com/OWASP/owasp-mstg.svg?branch=master)](https://travis-ci.com/OWASP/owasp-mstg)
+
+The checklists contained in the excel files allow a mapping between a given version of the [OWASP Mobile Security Testing Guide (MSTG)](https://github.com/OWASP/owasp-mstg "MSTG") and the [OWASP Mobile Application Verification Standard (MASVS)](https://github.com/OWASP/owasp-masvs "MASVS").
+
+The MSTG version element (Dashboard - row 13) in the excel file represent the version of mstg which the links in the excel file will lead to.
+
+Note that due to the continuous updating and restructuring of the MSTG, the links of different versions are not all identical. And changing the mstg version (Dashboard - row 13) may break the links in the excel file.
```
Comment (commjoen) [inline Checklists/README.md]: Can you please add the following:
```suggestion
Note that due to the continuous updating and restructuring of the MSTG, the links of different versions are not all identical. And changing the mstg version (Dashboard - row 13) may break the links in the excel file.

Newer and older versions of the Excel are released on a regular basis and can be found at [the release page](https://github.com/OWASP/owasp-mstg/releases "Releases").
```

**Pull request merged**